modules/KoreBuild.Tasks/Utilities/VsInstallerHelper.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Build.Utilities;
+using Newtonsoft.Json;
+
+namespace KoreBuild.Tasks.Utilities
+{
+    internal class VsInstallerHelper
+    { 
+        public static async Task<string> DownloadVsExe(TaskLoggingHelper log)
+        {
+            // TODO put this in an obj folder instead of temp?
+            var tempPath = Path.Combine(Path.GetTempPath(), "vs_enterprise.exe");
+
+            await DownloadFileHelper.DownloadFileAsync(uri: "https://aka.ms/vs/15/pre/vs_enterprise.exe",
```

Comment:
This is only available on corpnet. Switch this to https://aka.ms/vs/15/release/vs_enterprise.exe instead.